app/assets/stylesheets/application.css.scss

```diff
@@ -3,26 +3,167 @@
  */
 
 $bg-color: #f5f8fa;
-$navs-bg-color: #080808;
+$navs-bg-color: #1e1e1e;
 $white: #ffffff;
 $border: #e1e8ed;
+$navs-hover-color: #aaa;
+$menu-drop: #333;
+$menu-drop-hover: #333;
+$menu-disabled: #ddd;
+$black: #000;
+$form-input-glyph: #555;
+$form-input-bg-color: #f5f5f5;
+$form-input-button: #2a292a;
+$form-input-button-hover: #3a393a;
+$form-a-color: gray;
+$form-a-hover-color: #333;
+$form-login-gray-text: #9b9b9b;
+$btn-landing: rgba(255, 255, 255, 0.7);
+$btn-landing-hover: rgba(255, 255, 255, 0.9);
+$pages-title: #4a4a4a;
+$pages-text: #4a4a4a;
+$pages-anchor: #4a4a4a;
+$pages-anchor-hover: #8a8a8a;
+$features-background: #f5f5f5;
+$features-separator: #d8d8d8;
 
 [ng-cloak],
 .ng-cloak {
   visibility: hidden !important;
 }
 
-body[data-action="devise/sessions-new"], body[data-action="devise/sessions-create"] {
-  background: image-url('red.jpg') no-repeat center center fixed;
-  background-size: cover;
+html {
```

Comment:
For me it's ok, it's a way of declaring the global font-size (at the root element). I've just found some links that are maybe useful to get a deeper understanding on this:

https://css-tricks.com/almanac/selectors/r/root/
http://kyleschaeffer.com/development/css-font-size-em-vs-px-vs-pt-vs/
https://www.sitepoint.com/understanding-and-using-rem-units-in-css/